Library/NumberFormatter.swift

```diff
@@ -12,7 +12,10 @@ final class AttributedNumberFormatter: NumberFormatter {
   // MARK: - Attributed string
 
   override func attributedString(for obj: Any, withDefaultAttributes _: [NSAttributedString.Key: Any]? = nil) -> NSAttributedString? {
-    guard let number = obj as? NSNumber, let string = string(from: number) else { return nil }
+    guard
+      let number = obj as? NSNumber,
+      let string = string(from: number)?.replacingOccurrences(of: String.nbsp + String.nbsp, with: String.nbsp)
```

Comment:
This fixes tests and we already do this elsewhere. But do we want to do this here? 🤔